contracts/proxy/transparent/TransparentUpgradeableProxy.sol

```diff
@@ -36,75 +46,95 @@ contract TransparentUpgradeableProxy is ERC1967Proxy {
     }
 
     /**
-     * @dev Modifier used internally that will delegate the call to the implementation unless the sender is the admin.
+     * @dev If caller is the admin process the call internally, otherwise transparently fallback to the proxy behavior
      */
-    modifier ifAdmin() {
```

Comment:
Are there strong reasons to remove the `ifAdmin` modifier? I believe it may be better to keep it with a deprecation notice.

I imagine an argument for removing it is that users who may be extending the contract and using `ifAdmin` should also consider removing it in favor of the manual dispatch approach we follow here. But it doesn't feel necessary to force that change. If the function using `ifAdmin` has no arguments there is no issue.